### DISCUSSION TITLE
Change open direction of InputPicker

### DISCUSSION THREAD

**Discussion by PiousVenom** [rsuite/rsuite#2448] [Q&A]
Is there a way to have the InputPicker open above the input if the options would go below the bottom of the screen? Can't seem to find that option anywhere.

**Comment by simonguo** [comment]
Hi @PiousVenom 
The `placement` prop is supported on all Pickers.

```tsx
<InputPicker placement="topStart"  />
```
<img width="1071" alt="image" src="https://user-images.githubusercontent.com/1203827/164154818-dfb709b6-685f-4cb2-bbd6-261b4054778d.png">
